runtime/doc/lua.txt

```diff
@@ -1504,6 +1504,40 @@ schedule_wrap({cb})                                      *vim.schedule_wrap()*
         |vim.in_fast_event()|
 
 
+==============================================================================
+Lua module: inspector                                          *lua-inspector*
+
+inspect_pos({bufnr}, {row}, {col}, {filter})               *vim.inspect_pos()*
+    Get all the items at a given buffer position Can also be pretty printed
+    with *:Inspect!*
+
+    Parameters: ~
+      • {bufnr}   buffer|nil defaults to the current buffer
+      • {row}     (number|nil) 0-indexed row. Defaults to the current cursor
```

Comment:
To be more consistent with the wording used in api.txt:

```suggestion
      • {row}     (number|nil) row to inspect, 0-based. Defaults to row of the current cursor
```

And likewise below.